### PR TITLE
Temporarily disable gensim test

### PR DESCRIPTION
**Pull request by longdt219**
Since Gensim is not working with python 3.2, we temporarily remove it. It might be added back once we get answer from Gensim dev team.

### CODE REVIEW

**Comment by kmike** [comment]
I think that if we dropped 2.6 support it should be fine to drop 3.2 as well.

File: nltk/test/gensim_fixt.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+def setup_module(module):
+    from nose import SkipTest
+    try:
+        import gensim
+    except ImportError:
+        raise SkipTest("classify.doctest requires numpy")
```
Comment (kmike) [inline nltk/test/gensim_fixt.py]: the error message looks incorrect

**Comment by stevenbird** [comment]
Thanks @kmike. In that case, @longdt219 let's replace 3.2 with 3.3 on the CI server. I'll accept this PR but reinstate the gensim dependency.

File: nltk/test/gensim_fixt.py (hunk 1 above)
Comment (stevenbird) [inline nltk/test/gensim_fixt.py]: Thanks @kmike – I'll fix this

**Comment by longdt219** [comment]
Replace Python 3.2 by python3.3 through PR #1162

**Pull request merged**